metrics.go

```diff
@@ -258,6 +265,7 @@ func (m *metrics) doPost(url *url.URL, payload interface{}) (*http.Response, err
 	req.Header.Add("UNLEASH-APPNAME", m.options.appName)
 	req.Header.Add("UNLEASH-INSTANCEID", m.options.instanceId)
 	req.Header.Add("User-Agent", m.options.appName)
+	req.Header.Add("Unleash-Interval", fmt.Sprintf("%d", m.options.metricsInterval.Milliseconds()))
```

Comment:
it's weird that UNLEASH-INSTANCEID is all uppercase and this one is Unleash-Interval

Reply:
It is just industry standard to have headers as titled case.